pkg/tfbridge/names.go

```diff
@@ -102,8 +101,6 @@ func TerraformToPulumiName(name string, sch shim.Schema, ps *SchemaInfo, upper b
 	casingActivated := false // tolerate leading underscores
 	for i, c := range name {
 		if c == '_' && casingActivated {
-			// skip underscores and make sure the next one is capitalized.
-			contract.Assertf(!nextCap, "Unexpected duplicate underscore: %v", name)
```

Comment:
Nit: Might be nice to have a comment explaining that we will ignore duplicate `_`.

Reply:
Added